database-commons/src/main/java/io/cdap/plugin/db/CommonSchemaReader.java

```diff
@@ -120,4 +124,22 @@ public Schema getSchema(ResultSetMetaData metadata, int index) throws SQLExcepti
 
     return Schema.of(type);
   }
+
+  @Override
+  public boolean ignoreColumn(ResultSetMetaData metadata, int index) throws SQLException {
```

Comment:
Shouldn't we validate it instead of silently ignore?

Reply:
What kind of validation do we need here? This is default implementation of method that allows us to not include a column in the inferred schema. By default, all columns must be included. 

But there is a special case with MS SQL TIMESTAMP column and we [ignore it](https://github.com/vladhlinskiy/database-plugins/pull/5/files#diff-2a0c1456a046bf3d9ad08c09c6045021R61) for MS SQL Sink:
```
    @Override
    public boolean ignoreColumn(ResultSetMetaData metadata, int index) throws SQLException {
	  // Ignore 'TIMESTAMP' column in the output schema since values of this type are generated automatically and can
	  // not be set nor updated
	  return TIMESTAMP_TYPE_NAME.equalsIgnoreCase(metadata.getColumnTypeName(index));
    }
```